webservices/resources/legal.py

```diff
@@ -0,0 +1,30 @@
+import sqlalchemy as sa
+
+from flask_apispec import doc, marshal_with
+
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices.utils import use_kwargs
+from webargs import fields
+from pyelasticsearch import ElasticSearch
+from webservices.env import env
+
+
+@doc(
+    tags=['search'],
+    description=docs.LEGAL_SEARCH
+)
+
+class UniversalSearch(utils.Resource):
+    def __init__(self):
+        es_conn = env.get_credential('ES_CONN')
```

Comment:
* The `get_credential` method takes an optional default argument, so you can write `env.get_credential('ES_CONN', 'http://localhost:9200')`
* I'm guessing this will use the cloud.gov elastic service in real life, so it might be worth writing the configuration for that in this patch. Example at https://github.com/18F/atf-eregs/blob/master/atf_eregs/settings/prod.py#L33-L40.
* AFAICT, this is going to make a new `ElasticSearch` client every time a request gets dispatched to this view. I'm guessing that's not what we want to do.